packages/beacon-node/src/chain/bls/multithread/jobItem.ts

```diff
@@ -60,20 +61,31 @@ export function jobItemWorkReq(job: JobQueueItem, format: PointFormat): BlsWorkR
           message: set.signingRoot,
         })),
       };
-    case JobQueueItemType.sameMessage:
+    case JobQueueItemType.sameMessage: {
+      // validate signature = true, this is slow code on main thread so should only run with network thread mode (useWorker=true)
+      // For a node subscribing to all subnets, with 1 signature per validator per epoch it takes around 80s
+      // to deserialize 750_000 signatures per epoch
+      // cpu profile on main thread has 250s idle so this only works until we reach 3M validators
+      // However, for normal node with only 2 to 7 subnet subscriptions per epoch this works until 27M validators
+      // and not a problem in the near future
+      // this is monitored on v1.11.0 https://github.com/ChainSafe/lodestar/pull/5912#issuecomment-1700320307
+      const timer = metrics?.blsThreadPool.signatureDeserializationMainThreadDuration.startTimer();
+      const signatures = job.sets.map((set) => bls.Signature.fromBytes(set.signature, CoordType.affine, true));
+      if (timer) {
+        timer();
+      }
```

Comment:
can be written as below, please double check syntax may be wrong
```suggestion
      timer?.()
```